config/config_test.go

```diff
@@ -1553,6 +1554,18 @@ func TestOTLPSanitizeResourceAttributes(t *testing.T) {
 	})
 }
 
+func TestOTLPAllowUTF8(t *testing.T) {
+	t.Run("good config", func(t *testing.T) {
+		_, err := LoadFile(filepath.Join("testdata", "otlp_allow_utf8.good.yml"), false, promslog.NewNopLogger())
+		require.NoError(t, err)
```

Comment:
You should verify the outcome:
```suggestion
		c, err := LoadFile(filepath.Join("testdata", "otlp_allow_utf8.good.yml"), false, promslog.NewNopLogger())
		require.NoError(t, err)
		require.Equal(t, NoUTF8EscapingWithSuffixes, conf.OTLPConfig.TranslationStrategy)

```